diff --git a/idpselect_config.js b/idpselect_config.js
index 440f661..902ae31 100644
--- a/idpselect_config.js
+++ b/idpselect_config.js
@@ -24,7 +24,7 @@ function IdPSelectUIParms(){
     this.preferredIdP = null;        // Array of entityIds to always show
     this.hiddenIdPs = null;          // Array of entityIds to delete
     this.ignoreKeywords = false;     // Do we ignore the <mdui:Keywords/> when looking for candidates
-    this.showListFirst = false;      // Do we start with a list of IdPs or just the dropdown
+    this.showListFirst = true;      // Do we start with a list of IdPs or just the dropdown
     this.samlIdPCookieTTL = 730;     // in days
     this.setFocusTextBox = true;     // Set to false to supress focus 
     this.testGUI = false;
